lib/rules/display-name.js

```diff
@@ -70,6 +77,58 @@ module.exports = {
       return node.type === 'CallExpression' && argumentIsCallExpression && utils.isPragmaComponentWrapper(node);
     }
 
+    /**
+     * Checks if the node is a React.createContext call
+     * @param {ASTNode} node - The AST node being checked.
+     * @returns {Boolean} - True if node is a React.createContext call, false if not.
+     */
+    function isCreateContext(node) {
```

Comment:
this seems like a good idea to add to `Components.js`, so other rules could share it?

Reply:
Sure, a separate file is fine too, it just may need to be in the things provided to `Components.detect`